src/lib/ext-listen.js

```diff
@@ -0,0 +1,29 @@
+export async function getAllExtensions() {
+  const extensions = await browser.management.getAll();
+  return extensions.filter((extension) => extension.type !== 'theme');
+}
+
+export async function areExtsBeingMonitored() {
+  const { status } = await browser.runtime.sendMessage({
+    getMonitoringStatus: true,
```

Comment:
how about using messages like `{requestType: "getMonitoringStatus"}` instead of `{getMonitoringStatus: true}`?

Reply:
This is done. Thanks for the idea.